apps/comps/components/leaderboard-table.tsx

```diff
@@ -74,10 +79,11 @@ export function LeaderboardTable({
             >
               <TableCell className="xs:flex-row flex flex-col items-center gap-2 py-6">
                 <div className="flex gap-2">
-                  <ArrowUp size={20} className="text-green-500" />
-                  <span>3</span>
+                  {/* TODO: enable once we have official rankings & net change in rank */}
```

Comment:
same theme here as the comment below...since we cant show change in rank, we might as well hide it, for now